command/src/config.rs

```diff
@@ -523,6 +523,26 @@ pub struct HttpAppConfig {
 }
 
 impl HttpAppConfig {
```

Comment:
the modifications to `command/src/config.rs` are not needed. The `Config` object should not be modified after it has been created